freqtrade/constants.py

```diff
@@ -476,7 +480,9 @@
                 'outdated_offset': {'type': 'integer', 'minimum': 1},
                 'markets_refresh_interval': {'type': 'integer'},
                 'ccxt_config': {'type': 'object'},
-                'ccxt_async_config': {'type': 'object'}
+                'ccxt_async_config': {'type': 'object'},
+                'price_rounding_mode': {'type':    'integer', 'enum': PRICE_ROUND_MODES,
```

Comment:
i don't think that making this configurable is a good idea.
You'll get into the "how to" discussion very quickly.

maybe i'd like to round down on buys, and round up on sells ... - or "up on entries" "down on exits" (which is different from the prior in case of shorts).

Reply:
I agree, I also wouldn't add it as a configuration, and would change the default rounding to ROUND, instead of ROUND_UP, to avoid redundant configuration and discussions.

Do you think that it's okay to change the default rounding from ROUND_UP to ROUND?

I think this would simplify this PR a lot; with the exception of the tests that would be broken by this change :)